Sources/SourceKitLSP/Swift/DocumentFormatting.swift

```diff
@@ -215,3 +248,9 @@ extension SwiftLanguageService {
     return edits(from: snapshot, to: formattedString)
   }
 }
+
+private extension String {
+  var isNewline: Bool {
+    self == "\n" || self == "\r\n" || self == "\r"
+  }
```

Comment:
This property should already exist on `Character`. I would prefer to re-use that if possible instead of re-defining it.

Reply:
It appears this is not needed.

Reply:
I originally did that because i suspected it'll be more performant.
There are some code in this project that also avoid calling into `CharacterSet`-related functions.